spec/job_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'spec_helper'
+require 'sequenceserver/job'
+require 'sequenceserver/blast/job'
+require 'rack/test'
+
+# Test Job class from module SequenceServer
+module SequenceServer
+    describe 'blank Job' do
+        my_job = Job.new
+
+        it 'should have an id' do
+            expect(my_job.id).to match(/\A(urn:uuid:)?[\da-f]{8}-([\da-f]{4}-){3}[\da-f]{12}\z/i)
+        end
+
+        it 'should have a time of submission' do
+            expect(my_job.submitted_at).to be_kind_of(Time)
+        end
+
+        it 'should have an false exitstatus' do
+            expect(my_job.done?).to be_falsey
+        end
+
+        it 'should not be completed yet' do
+            expect(my_job.completed_at).to be(nil)
+        end
+
+        it 'should have a job directory' do
+            expect("#{__dir__}/dotdir/#{my_job.id}").not_to be_empty
+        end
+    end
+
+    describe 'BLAST Job' do
+
+        # Test Job class from module Blast and params
+        ENV['RACK_ENV'] = 'test'
+        include Rack::Test::Methods
+    
+        before do
+            SequenceServer.init(database_dir: "#{__dir__}/database/v5/sample")
+
+            @params = {
+                sequence: 'AGCTAGCTAGCT', 
+                databases: [Database.first.id],
+                method: (Database.first.id == 'protein' ? 'blastp' : 'blastn'),
+            }
+        end
+
+        context 'with params' do
+            let(:test_job){test_job = Job.create(@params)}
```

Comment:
Does this file pass rubocop check?

Reply:
I wanted to say yes. But I am not very sure since I was just expecting this when it passed all general checks here. Let me check this. I come back to this later.

Reply:
Rubocop checks are passing now, despite the very long describe block (should be under 25 lines). I checked with other files and we also have the problem there. For now I left it like that since it is not in the rubocop style guide and it is not disturbing regarding readability. If we would change that we should generally do that everywhere.